src/editor/Core/Ripgrep.re

```diff
@@ -0,0 +1,38 @@
+[@deriving show]
+type t = {search: string => list(string)};
+
+let mergeOptions = options =>
+  List.fold_left((accum, opt) => accum ++ " " ++ opt, "", options);
+
+let process = (rgPath, args) => {
+  let lines = ref([]);
+  /**
+     NOTE: this is run in parallel to the main program
+   */
+  let inChannel = mergeOptions([rgPath, ...args]) |> Unix.open_process_in;
```

Comment:
We found a couple options to fix the path-in-space issue:
- Use quotes around `rgPath`, ie `"\"" ++ rgPath ++ "\""`
- Use the `Rench` `ChildProcess.spawn` helper (`Unix.create_process` doesn't seem to have this issue)